src/custom/pages/App/index.tsx

```diff
@@ -10,6 +10,23 @@ import TermsAndConditions from 'pages/TermsAndConditions'
 import About from 'pages/About'
 import Faq from 'pages/Faq'
 import CowGame from 'pages/CowGame'
+import * as Sentry from '@sentry/react'
+import { Integrations } from '@sentry/tracing'
+
+const SENTRY_DSN = process.env.REACT_APP_SENTRY_DSN
+const SENTRY_TRACES_SAMPLE_RATE = process.env.REACT_APP_SENTRY_TRACES_SAMPLE_RATE
+
+if (SENTRY_DSN) {
+  Sentry.init({
+    dsn: process.env.REACT_APP_SENTRY_DSN,
+    integrations: [new Integrations.BrowserTracing()],
+
+    // Set tracesSampleRate to 1.0 to capture 100%
+    // of transactions for performance monitoring.
+    // We recommend adjusting this value in production
+    tracesSampleRate: SENTRY_TRACES_SAMPLE_RATE ? Number(SENTRY_TRACES_SAMPLE_RATE) : 1.0,
```

Comment:
Is there a rate limit or something?

Reply:
They make u pay if u pass some amount of errors. For now i left this that is the default, we just need to code with no bugs so there's no a lot of notifications :P 

Well see how many we get, and adjust